src/backend/functions/setup/s3-handler.js

```diff
@@ -34,19 +44,21 @@ module.exports = s3 => {
       ),
 
     removeFiles: () =>
-      listFiles({
-        Bucket: TO_BUCKET
-      }).then(result =>
-        Promise.all(
-          result.Contents.map(file => file.Key).map(file =>
-            deleteFile({
-              Bucket: TO_BUCKET,
-              Key: file
-            })
+      listFiles({ Bucket: TO_BUCKET }).then(result =>
```

Comment:
Starting to look a bit like the Pyramid of Doom here. ;)

Reply:
Refactored it a little but, should be better now with some `Promise.all` so everything should be quicker